src/maidsafe/vault/mpid_manager/mpid_manager.proto

```diff
@@ -1,4 +1,4 @@
-/*  Copyright 2012 MaidSafe.net limited
+/*  Copyright 2015 MaidSafe.net limited
```

Comment:
I don't think the copyright date needs to be updated unless there's a major rewrite of the file in question.